package.json

```diff
@@ -51,35 +52,37 @@
     "@vitest/coverage-c8": "^0.32.0",
     "@vitest/coverage-v8": "^1.0.4",
     "@vitest/ui": "^1.3.1",
-    "babel-jest": "29.4.3",
+    "babel-jest": "29.7.0",
     "cypress": "13.13.0",
     "esbuild": "0.19.2",
     "eslint": "8.57.0",
     "eslint-config-next": "14.2.3",
     "eslint-config-prettier": "9.0.0",
-    "eslint-plugin-cypress": "2.13.4",
-    "eslint-plugin-import": "2.27.5",
-    "eslint-plugin-jsx-a11y": "6.7.1",
-    "eslint-plugin-react": "7.32.2",
-    "eslint-plugin-react-hooks": "4.6.0",
+    "eslint-plugin-cypress": "^3.5.0",
+    "eslint-plugin-import": "2.31.0",
+    "eslint-plugin-jsx-a11y": "6.10.1",
+    "eslint-plugin-react": "7.35.0",
+    "eslint-plugin-react-hooks": "5.0.0",
     "gh-pages": "^5.0.0",
-    "jest": "29.4.3",
-    "jest-environment-jsdom": "29.4.3",
+    "jest": "29.7.0",
+    "jest-environment-jsdom": "29.7.0",
     "jest-environment-node": "^29.4.1",
+    "jiti": "2.4.2",
     "jsdom": "22.1.0",
     "json-bigint": "^1.0.0",
-    "nx": "19.5.6",
+    "jsonc-eslint-parser": "^2.1.0",
+    "nx": "20.5.0",
     "prettier": "^2.8.4",
     "prettier-plugin-organize-imports": "^3.2.4",
     "react-refresh": "^0.10.0",
     "supabase": "^1.178.2",
     "swc-loader": "0.1.15",
     "ts-jest": "29.1.0",
     "ts-node": "10.9.1",
-    "typescript": "5.5.4",
+    "typescript": "5.7.3",
```

Comment:
We could bump Typescript to 5.8.2; 5.8 doesn't have anything particularly useful though

Reply:
maybe next version bump i've suffered enough for this one.